app/server/models/community.model.js

```diff
@@ -42,24 +42,30 @@
  *          description: Users that have joined the community.
  *          items:
  *            - $ref: '#/components/schemas/User'
+ *        rules:
+ *          type: string
+ *          description: Community rules set by moderators
  *      required:
  *        - title
+ *        - description
+ *        - creator
+ *        - rules
  */
 
 const mongoose = require("mongoose");
 
 const Community = new mongoose.Schema(
   {
     title: { type: String, required: true, unique: true },
-    description: { type: String },
-    creator: { type: mongoose.Schema.Types.ObjectId, ref: "User" },
+    description: { type: String, required: true },
+    creator: { type: String },
```

Comment:
How come description is required but creator isn't?

Reply:
My bad. Thanks for catching that